.changeset/soft-socks-count.md

```diff
@@ -0,0 +1,22 @@
+---
+"@neo4j/graphql": patch
+---
+
+Add `version` to `cypherQueryOptions` in context to add a Cypher version with `CYPHER` before each query:
+
+```js
+{
+    cypherQueryOptions: {
+        addCypherVersionPrefix: true,
```

Comment:
Can we make this just `addVersionPrefix`? Kind of implied it's Cypher version given it's in `cypherQueryOptions`.